Realm.Shared/linq/RealmResultsVisitor.cs

```diff
@@ -117,6 +118,29 @@ private void AddSort(LambdaExpression lambda, bool isStarting, bool ascending)
         }
 
 
+        private RowHandle VisitElementAt(MethodCallExpression m)
+        {
+            Visit(m.Arguments.First());
+            var index = (int)ExtractConstantValue(m.Arguments.Last());
+
+            RowHandle row = null;
```

Comment:
I'd guess it was before you assigned it in one of the branches, or it could be a XS issue - their static analysis is not super-duper stable 😜